### ISSUE TITLE
Autoprefixer doesn’t support Node v0.10.37

### ISSUE THREAD

**Issue by dheenaINDIAN** [ai/autoprefixer-rails#140]
I am using 9.1.3 version
Getting error while integrating with bootstrap as,
 
"Autoprefixer doesn’t support Node v0.10.37. Update it. "

**Comment by ai** [comment]
Yes. It doesn’t support it because 0.10 doesn’t have security updates anymore. Please update it ASAP, you make your business under attack.

**Comment by cesc1989** [comment]
Hi @ai I think what @dheenaINDIAN says is that autoprefixer is using the "wrong" version of node.

In my case, I had to install a newer version (8.11.4) using nvm but I already had installed 0.46 via apt-get.

I tried to see if it was something the gem does or a dependency here https://github.com/ai/autoprefixer-rails/blob/master/lib/autoprefixer-rails/processor.rb#L159 and seems to be more of a thing from exec-js gem.

Anyway, I removed the one I've installed via apt-get and now the error's gone.
